spec/support/scenarios_helper.rb

```diff
@@ -207,6 +207,7 @@ def perform_billing
     Clock::SubscriptionsBillerJob.perform_later
     Clock::FreeTrialSubscriptionsBillerJob.perform_later
     perform_all_enqueued_jobs
+    perform_usage_update
```

Comment:
I'm adding this perform_billing because creating invoices influence `lifetime_usage.invoiced_usage_amount_cents` so lifetime_usage should be recomputed. We should keep scenarios as close as possible to real life.